jest.config.js

```diff
@@ -8,6 +8,8 @@ module.exports = {
     transform: {
         '^.+\\.jsx?$': 'babel-jest',
         '^.+/es/^.+$': 'babel-jest',
+        '^.+\\.ts?$': 'ts-jest',
+        '^.+\\.tsx?$': 'ts-jest',
```

Comment:
```suggestion
        '^.+\\.(ts|tsx)?$': 'ts-jest',
```
@niloo-fs I think we can do something like this, Not sure 😅

Reply:
That's weird but it is working too :))) I'll update this